examples/ex3.cpp

```diff
@@ -19,6 +20,16 @@
 //               ex3 -m ../data/mobius-strip.mesh -f 0.1
 //               ex3 -m ../data/klein-bottle.mesh -f 0.1
 //
+// Device sample runs:
+//               ex3 -m ../data/star.mesh -pa -d cuda
+//               ex3 -m ../data/star.mesh -pa -d raja-cuda
+//               ex3 -m ../data/star.mesh -pa -d occa-cuda
+//               ex3 -m ../data/star.mesh -pa -d raja-omp
+//               ex3 -m ../data/star.mesh -pa -d occa-omp
+//               ex3 -m ../data/star.mesh -pa -d ceed-cpu
+//               ex3 -m ../data/star.mesh -pa -d ceed-cuda
+//               ex3 -m ../data/beam-hex.mesh -pa -d cuda
```

Comment:
This line triggers error with CUDA. We should either use some template version, use lower `MAX_*` bounds (instead of using the ones in `forall.hpp`) that must be given directly to kernel - or comment it out.